app/models/game.rb

```diff
@@ -0,0 +1,8 @@
+class Game < ActiveRecord::Base
+
+  belongs_to :jedi_user_id, foreign_key: 'jedi_user_id', class_name: "User"
+  belongs_to :sith_user_id, foreign_key: 'sith_user_id', class_name: "User"
+  has_many :ships
```

Comment:
Try it like this:
```
belongs_to :jedi_user, foreign_key: 'jedi_user_id', class_name: "User"
belongs_to :sith_user, foreign_key: 'sith_user_id', class_name: "User"
```
ActiveRecord may be particular about this.  If you leave the belongs_to bit with the _id suffix at the end ActiveRecord with it's smart naming conventions may want to add another _id to the end even though you've already explicitly named the foreign_key.  Best to go with the flow of rails here and let ActiveRecord do it's thing.